main/ZgatewayRF.ino

```diff
@@ -50,7 +50,8 @@ void RFtoMQTTdiscovery(SIGNAL_SIZE_UL_ULL MQTTvalue) { //on the fly switch creat
                   subjectRFtoMQTT, switchRF[1], (char*)getUniqueId(switchRF[1], switchRF[2]).c_str(),
                   will_Topic, switchRF[3], switchRF[4],
                   switchRF[5], switchRF[6], switchRF[7],
-                  0, "", "", true, subjectMQTTtoRF);
+                  0, "", "", true, subjectMQTTtoRF,
+                  "", "", "", "");
```

Comment:
There is too much parameters, last one should be removed (for the diff below also)

Reply:
Are you sure ? I see 19 parameters in the new definition of the function and there is 19 here too

Reply:
Yes you are right